src/VisualStudio/Core/Def/Implementation/PullMemberUp/MainDialog/PullMemberUpDialogViewModel.cs

```diff
@@ -65,6 +65,7 @@ public BaseTypeTreeNodeViewModel SelectedDestination
                     foreach (var member in fields)
                     {
                         member.IsCheckable = !isInterface;
+                        member.HelpText = isInterface ? ServicesVSResources.Interface_cannot_have_field : string.Empty;
```

Comment:
Why do we have this additional check? How do we show this visually?

Reply:
This is because the text shouldn't exist if we don't show a tooltip. Right now, if there's no Explicit `AutomationProperties.HelpText` set it reverts to using the tooltip text. In the latest push, I modified so we set this to be tooltip instead of help text. 

We want that to reflect that a tooltip is not showing. In cases where it is, it's to notify users of why a checkbox is disabled. However, when the tooltip is disabled (`IsCheckable == true`) we don't want screen readers or a tooltip to say anything.

Reply:
Sounds good.